neural_compressor/adaptor/torch_utils/teq.py

```diff
@@ -334,32 +296,35 @@ def train(self, dataloader, train_steps=100, lr=1e-3, warmup_ratio=0.05,
                     optimizer.zero_grad()
                     lr_scheduler.step()
 
-                if global_steps == train_steps:
+                if global_steps >= train_steps:
                     break
 
         logger.info("finish training")
         self.model.eval()
         return None
 
     @torch.no_grad()
-    def quantize(self, scheme=None, quant_lm_head=False):
+    def quantize(self, quant_lm_head=False):
```

Comment:
if layer wise config has been supported, why do we still need quant_lm_head

Reply:
ok, remove it